editor/resources/meta.edn

```diff
@@ -670,10 +680,12 @@
   {:type :integer,
    :help "minimum API Level required for the application to run (android:minSdkVersion)",
    :default 19,
+   :minimum 19,
```

Comment:
Use separate `minimum` and `maximum` instead of `range`, so now we can specify just minimum separatly